src/cascade/dismod/db/metadata.py

```diff
@@ -495,6 +495,17 @@ class DataSim(Base):
     data_sim_delta = Column(Float(), nullable=False)
 
 
+class PriorSim(Base):
```

Comment:
We weren't using this because we were calling simulate and then predict, so this table was implicitly there. Now we read it because the code pulls a single model and data simulation out of the database.